src/main/java/org/usfirst/frc4079/RobotBuilderProject1/commands/GamepadRunIntake.java

```diff
@@ -0,0 +1,56 @@
+/*----------------------------------------------------------------------------*/
+/* Copyright (c) 2018 FIRST. All Rights Reserved.                             */
+/* Open Source Software - may be modified and shared by FRC teams. The code   */
+/* must be accompanied by the FIRST BSD license file in the root directory of */
+/* the project.                                                               */
+/*----------------------------------------------------------------------------*/
+
+package org.usfirst.frc4079.RobotBuilderProject1.commands;
+
+import org.usfirst.frc4079.RobotBuilderProject1.Robot;
+
+import edu.wpi.first.wpilibj.command.Command;
+
+public class GamepadRunIntake extends Command {
+  public GamepadRunIntake() {
+    // Use requires() here to declare subsystem dependencies
+    // eg. requires(chassis);
+    requires(Robot.intake);
+  }
+
+  // Called just before this Command runs the first time
+  @Override
+  protected void initialize() {
+  }
+
+  // Called repeatedly when this Command is scheduled to run
+  @Override
+  protected void execute() {
+    if(Robot.oi.gamePad.getRightBumper()) {
+        Robot.intake.run(0.4);
```

Comment:
Nit: For this number and the one below (`0.4` and `-0.3`), please extract them into a constant (in the current class or in the `Constants` class), to avoid [magic numbers](https://en.wikipedia.org/wiki/Magic_number_(programming)). You can name them something like `INTAKE_FORWARD_POWER` or something more relevant.